vsintegration/src/FSharp.VS.FSI/sessions.fs

```diff
@@ -298,6 +326,9 @@ type FsiSession(sourceFile: string) =
                 with _ -> () // if writing or flushing fails then just give up on this F# Interactive session
             })
 
+    do if usingNetCore then
+        inputQueue.Post($"""System.IO.File.WriteAllText(@"{trueProcessIdFile}", string (System.Diagnostics.Process.GetCurrentProcess().Id));;""")
```

Comment:
This is used in Visual Studio to get the process ID from F# Interactive

Reply:
Will the pid file be cleaned up or we just leave it?